controllers/backstage_deployment.go

```diff
@@ -302,3 +304,16 @@ func (r *BackstageReconciler) validateAndUpdatePsqlSecretRef(backstage bs.Backst
 
 	return nil
 }
+
+func (r *BackstageReconciler) setDefaultDeploymentImage(deployment *appsv1.Deployment) {
+	for i, c := range deployment.Spec.Template.Spec.InitContainers {
```

Comment:
The same operation is done on several containers, coming from different parts of the Pod Template Spec.
There is a `visitContainers` function (accepting a `ContainerVisitor` function) that can be used for that specific purpose, I think. Maybe it can be used here instead?

Reply:
Sounds good. Done